app/views/questions/search.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container container_body">
+  <% if @result.count == 0 %>
+      <div class="row">
+        <h3> NO Record Found</h3>
+  <% else %>
+    <% @result.each do |q| %>
+      <%= link_to question_path(q) do %>
+        <div class="row">
+          <div class="col-xs-1 answer">
+            <div class="answer_content">
+              <%= q.answers.count %>
+            </div>
+            <div class="answer_content">
+              answers
+            </div>
+          </div>
+          <div class="col-xs-11 question">
```

Comment:
this whole view is repeated and duplicate of index page. use the same view on both places.